<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>593</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>791</width>
      <height>551</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Query</string>
     </attribute>
     <widget class="QComboBox" name="toQuery">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>20</y>
        <width>251</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>111</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Device to Query:</string>
      </property>
     </widget>
     <widget class="QGroupBox" name="groupBox">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>50</y>
        <width>761</width>
        <height>471</height>
       </rect>
      </property>
      <property name="title">
       <string>Results</string>
      </property>
      <widget class="QGroupBox" name="groupBox_2">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>50</y>
         <width>191</width>
         <height>411</height>
        </rect>
       </property>
       <property name="title">
        <string>Performance [W]</string>
       </property>
       <widget class="QLabel" name="qPassiveTotal">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>30</y>
          <width>141</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Passive Total:</string>
        </property>
       </widget>
       <widget class="QLabel" name="qActiveTotal">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>60</y>
          <width>141</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Active Total:</string>
        </property>
       </widget>
       <widget class="QLabel" name="qActiveDissipation">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>90</y>
          <width>141</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Active Dissipation:</string>
        </property>
       </widget>
       <widget class="QLabel" name="qCPUPower">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>120</y>
          <width>141</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>CPU Power:</string>
        </property>
       </widget>
       <widget class="QLabel" name="qGPUPower">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>150</y>
          <width>141</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>GPU Power:</string>
        </property>
       </widget>
       <widget class="QLabel" name="qRoSPower">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>180</y>
          <width>141</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>RoS Power:</string>
        </property>
       </widget>
      </widget>
      <widget class="QGroupBox" name="groupBox_3">
       <property name="geometry">
        <rect>
         <x>280</x>
         <y>50</y>
         <width>191</width>
         <height>411</height>
        </rect>
       </property>
       <property name="title">
        <string>Thermal [C]</string>
       </property>
       <widget class="QLabel" name="label_9">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>30</y>
          <width>141</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>CPU Max Temp:</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_10">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>60</y>
          <width>141</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>GPU Max Temp:</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_11">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>90</y>
          <width>141</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Keyboard Max Temp:</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_12">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>120</y>
          <width>141</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Backcover Max Temp:</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_13">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>150</y>
          <width>141</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Fan Inlet Air Temp:</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_14">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>180</y>
          <width>141</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Fan Outlet Air Temp:</string>
        </property>
       </widget>
      </widget>
      <widget class="QGroupBox" name="groupBox_4">
       <property name="geometry">
        <rect>
         <x>520</x>
         <y>50</y>
         <width>191</width>
         <height>411</height>
        </rect>
       </property>
       <property name="title">
        <string>Dimensions [mm]</string>
       </property>
       <widget class="QLabel" name="label_15">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>30</y>
          <width>141</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Device Width:</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_16">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>60</y>
          <width>141</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Device Thickness:</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_17">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>90</y>
          <width>141</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Device Weight:</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_18">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>120</y>
          <width>141</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Acoustics:</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_19">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>150</y>
          <width>141</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Fan Dimensions:</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_20">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>180</y>
          <width>141</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>RoS Power:</string>
        </property>
       </widget>
      </widget>
      <widget class="QLabel" name="queriedDevice">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>20</y>
         <width>191</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Device: </string>
       </property>
      </widget>
     </widget>
     <widget class="QPushButton" name="queryButton">
      <property name="geometry">
       <rect>
        <x>400</x>
        <y>20</y>
        <width>101</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>Query</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Compare</string>
     </attribute>
     <widget class="QComboBox" name="toCompare">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>20</y>
        <width>251</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>111</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Device to Compare:</string>
      </property>
     </widget>
     <widget class="QPushButton" name="addDevice">
      <property name="geometry">
       <rect>
        <x>400</x>
        <y>20</y>
        <width>80</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>Add</string>
      </property>
     </widget>
     <widget class="QPushButton" name="clearDevices">
      <property name="geometry">
       <rect>
        <x>500</x>
        <y>20</y>
        <width>80</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>Clear</string>
      </property>
     </widget>
     <widget class="QLabel" name="dev1">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>50</y>
        <width>201</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Device 1:</string>
      </property>
     </widget>
     <widget class="QLabel" name="dev2">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>50</y>
        <width>221</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Device 2:</string>
      </property>
     </widget>
     <widget class="QLabel" name="dev3">
      <property name="geometry">
       <rect>
        <x>480</x>
        <y>50</y>
        <width>221</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Device 3: </string>
      </property>
     </widget>
     <widget class="QTableWidget" name="compTable">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>80</y>
        <width>771</width>
        <height>401</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="exportTable">
      <property name="geometry">
       <rect>
        <x>420</x>
        <y>490</y>
        <width>141</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>Export Comparison Table</string>
      </property>
     </widget>
     <widget class="QPushButton" name="genTable">
      <property name="geometry">
       <rect>
        <x>600</x>
        <y>20</y>
        <width>101</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>Generate Table</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="exportfile">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>490</y>
        <width>113</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_21">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>490</y>
        <width>111</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Export File Name:</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_3">
     <attribute name="title">
      <string>Benchmark</string>
     </attribute>
     <widget class="QPushButton" name="benchmarkRun">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>490</y>
        <width>171</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>Benchmark</string>
      </property>
     </widget>
     <widget class="QPushButton" name="benchmarkClear">
      <property name="geometry">
       <rect>
        <x>400</x>
        <y>490</y>
        <width>171</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>Clear</string>
      </property>
     </widget>
     <widget class="QGroupBox" name="groupBox_5">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>80</y>
        <width>171</width>
        <height>391</height>
       </rect>
      </property>
      <property name="title">
       <string>Performance [W]</string>
      </property>
      <widget class="QLineEdit" name="b1_2">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>60</y>
         <width>61</width>
         <height>21</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="b1_1">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>30</y>
         <width>61</width>
         <height>21</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_3">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>30</y>
         <width>81</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Passive Total</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_4">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>60</y>
         <width>81</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Active Total</string>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_6">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>10</y>
        <width>451</width>
        <height>61</height>
       </rect>
      </property>
      <property name="title">
       <string>Device</string>
      </property>
      <widget class="QLineEdit" name="benchmarkDevice">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>30</y>
         <width>421</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>Samsung Smart Fridge</string>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_7">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>80</y>
        <width>171</width>
        <height>391</height>
       </rect>
      </property>
      <property name="title">
       <string>Performance</string>
      </property>
      <widget class="QLineEdit" name="b2_2">
       <property name="geometry">
        <rect>
         <x>102</x>
         <y>60</y>
         <width>61</width>
         <height>21</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="b2_1">
       <property name="geometry">
        <rect>
         <x>102</x>
         <y>30</y>
         <width>61</width>
         <height>21</height>
        </rect>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_8">
      <property name="geometry">
       <rect>
        <x>400</x>
        <y>80</y>
        <width>171</width>
        <height>391</height>
       </rect>
      </property>
      <property name="title">
       <string>Performance</string>
      </property>
      <widget class="QLineEdit" name="b3_2">
       <property name="geometry">
        <rect>
         <x>102</x>
         <y>60</y>
         <width>61</width>
         <height>21</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="b3_1">
       <property name="geometry">
        <rect>
         <x>102</x>
         <y>30</y>
         <width>61</width>
         <height>21</height>
        </rect>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_9">
      <property name="geometry">
       <rect>
        <x>590</x>
        <y>80</y>
        <width>171</width>
        <height>391</height>
       </rect>
      </property>
      <property name="title">
       <string>Performance</string>
      </property>
      <widget class="QLineEdit" name="b4_2">
       <property name="geometry">
        <rect>
         <x>102</x>
         <y>60</y>
         <width>61</width>
         <height>21</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="b4_1">
       <property name="geometry">
        <rect>
         <x>102</x>
         <y>30</y>
         <width>61</width>
         <height>21</height>
        </rect>
       </property>
      </widget>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuBenchmark_Data_Query">
    <property name="title">
     <string>Benchmarking Data Query</string>
    </property>
   </widget>
   <addaction name="menuBenchmark_Data_Query"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
